src/main/java/org/primeframework/mvc/cors/CORSFilter.java

```diff
@@ -304,7 +336,20 @@ private CORSRequestType checkRequestType(HTTPRequest request, String origin) {
    * @return true if this request should be excluded from CORS
    */
   private boolean excludedRequestURI(final String requestURI) {
-    return excludedPathPattern != null && excludedPathPattern.matcher(requestURI).find();
+    if (excludedPathPattern != null) {
+      return excludedPathPattern.matcher(requestURI).find();
+    }
+    else if (includedPathPattern != null) {
+      return !includedPathPattern.matcher(requestURI).find();
+    }
+    else if (includeUriChecker != null) {
+      return !includeUriChecker.test(requestURI);
+    }
+    else if (excludeUriChecker != null) {
+      return excludeUriChecker.test(requestURI);
+    }
+    // we're not using any of the functionality
+    return false;
```

Comment:
- `else` should be on the previous line. 
- Add a space after the last curly bracket before your `return` statement.

Reply:
Done